include/deal.II/distributed/cell_data_transfer.h

```diff
@@ -186,7 +192,7 @@ namespace parallel
         }
 
         /**
-         * @copydoc dealii::CoarseningStrategies::mean
+         * @copydoc dealii::CoarseningStrategies::mean()
```

Comment:
oh, that helps? I had stared at this for a few minutes and couldn't figure out what doxygen was missing...